core/src/datayoga_core/utils.py

```diff
@@ -79,11 +79,16 @@ def get_resource_path(relative_path: str) -> str:
         return path.join(os.path.dirname(__file__), "resources", relative_path)
 
 
-def split_field(field: str) -> List[str]:
-    return re.split(r"(?<!\\)\.", field)
+def split_field(field: str, *, __expression=re.compile(r"(?<!\\)\.")) -> List[str]:
+    r"""Split string by dots like str.split(), but allows escape a dot by the backslash.
```

Comment:
r"""Splits a string by dots, similar to `str.split()`, but allows escaping a dot with a backslash (\).

Reply:
✅